pkg/collector/python/init.go

```diff
@@ -361,7 +350,7 @@ func resolvePythonExecPath(pythonVersion string, ignoreErrors bool) (string, err
 	// don't want to use the default version (aka "python") but rather "python2" or
 	// "python3" based on the configuration. Also on some Python3 platforms there
 	// are no "python" aliases either.
-	interpreterBasename := "python" + pythonVersion
+	interpreterBasename := "python3"
```

Comment:
Should we move this to a constant?

Reply:
That sounds sensible to do, although I'd rather let someone from the owning team do so (it might also be worth waiting a bit for other variables such as Python2/3_HOME to be removed as well